mk/uvmt/xrun.mk

```diff
@@ -252,6 +252,9 @@ XRUN_COMP_FLAGS += -nowarn CGNSWA
 # Value Parameters without default values
 XRUN_COMP_FLAGS += -setenv CADENCE_ENABLE_AVSREQ_44905_PHASE_1=1
 
+# Type Parameters without default values
+XRUN_COMP_FLAGS += -setenv CADENCE_ENABLE_AVSREQ_63188_PHASE_1=1
```

Comment:
Same as the above "Value Parameters" option, but for "Type Parameters".

This makes Xcelium behave LRM-compliant.
(Their explanation for why it is not default is that _"users may prefer that this issue should be reported during compilation stage and not later during elab"_, but it actually means that some checking will never happen at all.)

Reply:
Hiding, as reviewers probably have read it already.